### PR TITLE
Em 5427 update checkstyle

### PR DESCRIPTION
**Pull request by Jefex7**
### JIRA link (if applicable) ###

[https://tools.hmcts.net/jira/browse/EM-5427](url)

### Change description ###

Update Checksyle

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/em/annotation/config/security/SecurityConfiguration.java [hunk 1]
```diff
@@ -62,7 +66,7 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
                 .sessionManagement()
                 .sessionCreationPolicy(SessionCreationPolicy.STATELESS)
                 .and()
-                .authorizeRequests()
+                .authorizeHttpRequests()
```
Comment (mdayican) [inline src/main/java/uk/gov/hmcts/reform/em/annotation/config/security/SecurityConfiguration.java:69]: is this related check style ?

File: src/test/java/uk/gov/hmcts/reform/em/annotation/rest/CommentResourceIntTest.java [hunk 2]
```diff
@@ -26,20 +27,25 @@
...
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 /**
  * Test class for the CommentResource REST controller.
  *
  * @see CommentResource
  */
+
+@SuppressWarnings({"AbbreviationAsWordInName"})
```
Comment (mdayican) [inline src/test/java/uk/gov/hmcts/reform/em/annotation/rest/CommentResourceIntTest.java:48]: same here, we can try to fix it instead of suppressing.

File: src/test/java/uk/gov/hmcts/reform/em/annotation/rest/errors/ExceptionTranslatorIntTest.java [hunk 3]
```diff
@@ -13,14 +13,17 @@
...
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
 /**
  * Test class for the ExceptionTranslator controller advice.
  *
  * @see ExceptionTranslator
  */
 @RunWith(SpringRunner.class)
+@SuppressWarnings({"AbbreviationAsWordInName"})
```
Comment (mdayican) [inline src/test/java/uk/gov/hmcts/reform/em/annotation/rest/errors/ExceptionTranslatorIntTest.java:26]: same here, we can try to fix it instead of suppressing.

File: src/main/java/uk/gov/hmcts/reform/em/annotation/config/security/SecurityConfiguration.java (hunk 1 above)
Comment (Jefex7) [reply src/main/java/uk/gov/hmcts/reform/em/annotation/config/security/SecurityConfiguration.java:69]: authorizeRequets was deprecated

File: src/test/java/uk/gov/hmcts/reform/em/annotation/BaseTest.java [hunk 4]
```diff
@@ -46,7 +46,7 @@ public abstract class BaseTest {
     @Before
     public void setupMocks() {
 
-        MockitoAnnotations.initMocks(this);
+        MockitoAnnotations.openMocks(this);
```
[the remark this answers is not in the document]
Comment (Jefex7) [reply src/test/java/uk/gov/hmcts/reform/em/annotation/BaseTest.java:49]: initMocks is deprecated

File: src/test/java/uk/gov/hmcts/reform/em/annotation/rest/BookmarkResourceIntTest.java [hunk 5]
```diff
@@ -46,6 +46,7 @@
  *
  * @see BookmarkResource
  */
+@SuppressWarnings({"AbbreviationAsWordInName"})
```
[the remark this answers is not in the document]
Comment (mdayican) [reply src/test/java/uk/gov/hmcts/reform/em/annotation/rest/BookmarkResourceIntTest.java:49]: lets do them in separate pr ?

File: src/test/java/uk/gov/hmcts/reform/em/annotation/rest/CommentResourceIntTest.java (hunk 2 above)
Comment (Jefex7) [reply src/test/java/uk/gov/hmcts/reform/em/annotation/rest/CommentResourceIntTest.java:48]: no problem

File: src/test/java/uk/gov/hmcts/reform/em/annotation/rest/BookmarkResourceIntTest.java (hunk 5 above)
[the remark this answers is not in the document]
Comment (Jefex7) [reply src/test/java/uk/gov/hmcts/reform/em/annotation/rest/BookmarkResourceIntTest.java:49]: Okay

File: src/main/java/uk/gov/hmcts/reform/em/annotation/config/security/SecurityConfiguration.java (hunk 1 above)
Comment (mdayican) [reply src/main/java/uk/gov/hmcts/reform/em/annotation/config/security/SecurityConfiguration.java:69]: lets do them in separate pr ?

File: src/test/java/uk/gov/hmcts/reform/em/annotation/rest/errors/ExceptionTranslatorIntTest.java (hunk 3 above)
Comment (Jefex7) [reply src/test/java/uk/gov/hmcts/reform/em/annotation/rest/errors/ExceptionTranslatorIntTest.java:26]: okay

File: src/test/java/uk/gov/hmcts/reform/em/annotation/BaseTest.java (hunk 4 above)
[the remark this answers is not in the document]
Comment (mdayican) [reply src/test/java/uk/gov/hmcts/reform/em/annotation/BaseTest.java:49]: lets do them in separate pr ?

File: src/main/java/uk/gov/hmcts/reform/em/annotation/config/security/SecurityConfiguration.java (hunk 1 above)
Comment (Jefex7) [reply src/main/java/uk/gov/hmcts/reform/em/annotation/config/security/SecurityConfiguration.java:69]: okay I will revert the commit

File: src/test/java/uk/gov/hmcts/reform/em/annotation/BaseTest.java (hunk 4 above)
[the remark this answers is not in the document]
Comment (Jefex7) [reply src/test/java/uk/gov/hmcts/reform/em/annotation/BaseTest.java:49]: okay I will revert the commit

**Review by mdayican — APPROVED** [review]
